src/Mantasflowers.Persistence/DatabaseContext.cs

```diff
@@ -46,12 +46,29 @@ public DatabaseContext(DbContextOptions<DatabaseContext> options)
 
         public DbSet<Feedback> Feedback { get; set; }
 
+        public DbSet<HashMap> HashMap { get; set; }
+
         // TODO: future TODO and aren't used by anything
         // public DbSet<Supplier> Suppliers { get; set; }
         // public DbSet<Warehousing> Warehousing { get; set; }
 
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
+            modelBuilder.Entity<HashMap>(
+                e =>
+                {
+                    e.Property(p => p.PasswordHash)
+                        .HasMaxLength(300)
+                        .IsRequired();
+
+                    e.HasOne(p => p.Order)
+                        .WithOne();
```

Comment:
I think you can add cascading rules for this. When order is deleted, this should be deleted as well.

Reply:
Cascade is the default behavior. Have a look at a [DatabaseContextModelSnapshot.cs](https://github.com/mantasflowers/Mantasflowers.Backend/blob/M3-67_payment_order_flow_api/src/Mantasflowers.Persistence/Migrations/DatabaseContextModelSnapshot.cs)
```cs
modelBuilder.Entity("Mantasflowers.Domain.Entities.HashMap", b =>
    {
        b.HasOne("Mantasflowers.Domain.Entities.Order", "Order")
            .WithOne()
            .HasForeignKey("Mantasflowers.Domain.Entities.HashMap", "OrderId")
            .OnDelete(DeleteBehavior.Cascade)
            .IsRequired();

        b.Navigation("Order");
    });
```

Reply:
I would rather prefer having it explicit because if I need to know the behavior I'd rather look at human written code than something EF Core auto generated.